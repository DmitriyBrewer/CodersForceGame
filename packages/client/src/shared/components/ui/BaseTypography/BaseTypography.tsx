import { Typography } from '@mui/material'

const BaseTypography = Typography

export default BaseTypography
