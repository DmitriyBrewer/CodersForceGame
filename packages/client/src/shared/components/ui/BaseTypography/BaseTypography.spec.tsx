import { render, screen } from '@testing-library/react'

import '@testing-library/jest-dom'
import BaseTypography from './BaseTypography'

describe('<BaseTypography />', () => {
  const handleClick = jest.fn()

  it('Snapshot is correct', () => {
    const { asFragment } = render(<BaseTypography onClick={handleClick} />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('Renders correctly with default props', () => {
    render(<BaseTypography />)
    const button = screen.getByRole('button')

    expect(button).toMatchSnapshot()
  })
})
