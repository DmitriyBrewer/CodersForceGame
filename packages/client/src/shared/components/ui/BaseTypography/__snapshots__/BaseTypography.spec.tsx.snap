// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaseTypography /> Renders correctly with default props 1`] = `
<p
  class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
/>
`;

exports[`<BaseTypography /> Snapshot is correct 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
  />
</DocumentFragment>
`;
